src/manifest/dbtWorkspaceFolder.ts

```diff
@@ -52,12 +52,25 @@ export class DBTWorkspaceFolder implements Disposable {
         `**/{${DBTProject.DBT_MODULES.join(",")}}`,
       ),
     );
-    const projectFiles = dbtProjectFiles
+    var projectFiles = dbtProjectFiles
```

Comment:
Please never do `var`, rather keep on using `const` and apply the filter directly onto dbtProjectFiles inlining the same logic.

Reply:
yup, good callout. I'm not used to typescript 😁